js/script.js

```diff
@@ -0,0 +1,55 @@
+const inp = document.querySelector("input");
+const textForTyping = document.querySelector(".text-for-typing")
+const text = `I had a painting teacher once tell me that, the difference between a good painting and a great painting, is typically five strokes. And they're usually the five boldest strokes in the painting. The question, of course, is which five strokes?`
+
+//To prevent copying text during the game
+textForTyping.classList.add("noselect")
+
+//Split the text into letters and put it in a  <span> tag
+const characters = text.split('').map((char) => {
+    const span = document.createElement("span")
+    span.innerText = char;
+    textForTyping.appendChild(span)
+    return span
+});
+
+//Add cursor to first letter, then move it with user input
+let charIndex = 0;
+let isCorrect;
```

Comment:
It would be good to give an initial value to the boolean flag
```suggestion
let isCorrect = false;
```